### PR TITLE
Release v2.0.0rc1

### PR DESCRIPTION
**Pull request by gsnider2195**
- [x] bump pyproject.toml
- [x] towncrier

### CODE REVIEW

File: nautobot/ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "nautobot",
-    "version": "2.0.0-beta.2",
+    "version": "2.0.0-rc1",
```
Comment (bryanculver) [inline nautobot/ui/package.json:3]:
```suggestion
    "version": "2.0.0-rc.1",
```

File: nautobot/ui/package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     "name": "nautobot",
-    "version": "2.0.0-beta.2",
+    "version": "2.0.0-rc1",
     "lockfileVersion": 3,
     "requires": true,
     "packages": {
         "": {
             "name": "nautobot",
-            "version": "2.0.0-beta.2",
+            "version": "2.0.0-rc1",
```
Comment (bryanculver) [inline nautobot/ui/package-lock.json:9]:
```suggestion
            "version": "2.0.0-rc.1",
```

File: nautobot/ui/package-lock.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
     "name": "nautobot",
-    "version": "2.0.0-beta.2",
+    "version": "2.0.0-rc1",
```
Comment (bryanculver) [inline nautobot/ui/package-lock.json:3]:
```suggestion
    "version": "2.0.0-rc.1",
```

**Review by bryanculver — APPROVED** [review]

File: pyproject.toml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "2.0.0-beta.2"
+version = "2.0.0rc1"
```
Comment (HanlinMiao) [inline pyproject.toml:5]: Do we need the `-` between? 
```suggestion
version = "2.0.0-rc1"
```

**Review by HanlinMiao — APPROVED** [review]

**Pull request merged**